components/plan-build/bin/hab-plan-build.sh

```diff
@@ -922,11 +923,12 @@ _resolve_scaffolding_dependencies() {
   scaff_build_deps=()
   scaff_build_deps_resolved=()
 
+  # shellcheck disable=2066
```

Comment:
```
In ./components/plan-build/bin/hab-plan-build.sh line 926:
  for dep in "${pkg_scaffolding}"; do
             ^-- SC2066: Since you double quoted this, it will not word split, and the loop will only run once.
```
The quotes should almost certainly be removed, but I wasn't sure about the circumstances where `pkg_scaffolding` would have multiple values to ensure they'd split correctly.